gctf/protocols/protocol_gctf.py

```diff
@@ -440,16 +490,20 @@ def _argsGctf(self):
         self._args += "--convsize %d " % self.convsize.get()
         self._args += "--do_Hres_ref %d " % (1 if self.doHighRes else 0)
 
-        if gctf.Plugin.getActiveVersion() == '0.50':
-            self._args += "--do_basic_rotave %d " % (1 if self.doBasicRotave else 0)
-        else:
-            self._args += "--EPA_oversmp %d " % self.EPAsmp.get()
+        if self._isLatestVersion():
```

Comment:
In my experience with other plugins, isLatestVersion is a tricky condition, because when there is a new version, the current "latest" will not be the "lastest" and in some case maybe the same applies, but not in other cases...can we change this validation to a more precise one (or just more specific) ?